src/main/java/com/gooddata/executeafm/result/AttributeHeaderItem.java

```diff
@@ -30,18 +32,11 @@
      */
     @JsonCreator
     public AttributeHeaderItem(@JsonProperty("name") final String name, @JsonProperty("uri") final String uri) {
-        this.name = name;
+        super(name);
         this.uri = uri;
```

Comment:
I'd put `notEmpty` there

Reply:
it's in`super`

Reply:
ehh it's not but it should be - will fix it there